manifests/profile/hathitrust/apache.pp

```diff
@@ -27,6 +27,9 @@
     fact_for($nodename, 'networking')['ip']
   }
 
+  $staff_networks = lookup('hathitrust::networks::staff').flatten.unique.map |$network| {
+    $network['block']
```

Comment:
This should be `ip ${network['block']}` so we end up with e.g. `Require ip 10.1.2.0/24` in the Apache config